contracts/libraries/FVMLib.sol

```diff
@@ -37,7 +37,6 @@ bytes1 constant UNKNOWN = 0x00;
 bytes1 constant ALLOCATE = 0x01;
 bytes1 constant UNSET02 = 0x02;
 bytes1 constant DEALLOCATE = 0x03;
-bytes1 constant CLAIM = 0x04;
```

Comment:
Perhaps it would be more consistent to leave a `bytes1 constant UNSET04 = 0x04;` here instead?

Reply:
Yes, you're right.